exercises/practice/all-your-base/.approaches/introduction.md

```diff
@@ -0,0 +1,68 @@
+# Introduction
+
+The process of the _all your base_ exercise can be divided into  up to four parts:
```

Comment:
Here and in the rest of the PR: "All Your Base" is the title of the exercise, I think it should be capitalized and either put in italics (like here) or in quotes.